mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -250,44 +250,40 @@ def _load_mesh(self):  # {{{
             refTopDepth, refLayerThickness
         # }}}
 
-    def _cache_velocity_climatologies(self):  # {{{
+    def _compute_velocity_climatologies(self):  # {{{
         '''compute yearly velocity climatologies and cache them'''
 
-        variableList = ['avgNormalVelocity',
-                        'avgVertVelocityTop']
```

Comment:
This may actually be the more relevant function:
https://github.com/MPAS-Dev/MPAS-Analysis/blob/develop/mpas_analysis/shared/generalized_reader/generalized_reader.py#L379